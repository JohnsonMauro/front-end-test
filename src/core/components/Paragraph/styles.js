import styled from '@emotion/styled';

const ParagraphStyles = styled.p`
  font-family: ${({ theme }) => theme.fonts.Arial};
  margin: 8px 0;
`;

export { ParagraphStyles };
