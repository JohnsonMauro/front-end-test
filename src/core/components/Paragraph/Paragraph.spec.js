import React from 'react';
import { render } from '@testing-library/react';

import GlobalsContainer from '@test/GlobalsContainer';
import Paragraph from './index';

describe('<Paragraph />', () => {
  it('Renders the Paragraph component', () => {
    const { container } = render(<Paragraph>Sample</Paragraph>, {
      wrapper: GlobalsContainer,
    });
    expect(container).toMatchSnapshot();
  });
});
