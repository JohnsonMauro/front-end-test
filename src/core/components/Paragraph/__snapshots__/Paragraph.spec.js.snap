// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paragraph /> Renders the Paragraph component 1`] = `
.emotion-0 {
  margin: 8px 0;
}

<div>
  <p
    class="emotion-0 emotion-1"
  >
    Sample
  </p>
</div>
`;
